protocol/src/index.ts

```diff
@@ -651,6 +651,21 @@ export namespace Protocol {
     '|controlshtml|': readonly ['controlshtml', HTML];
     '|fieldhtml|': readonly ['fieldhtml', HTML];
     '|debug|': readonly ['debug', Message];
+    '|deinit|': readonly ['deinit'];
+    '|selectorhtml|': readonly ['selectorhtml', string, HTML];
+    '|refresh|': readonly ['refresh'];
+    '|tempnotify|': readonly ['tempnotify', string, Message, ...[] | [Message] | [Message, string]];
+    '|tempnotifyoff|': readonly ['tempnotifyoff', string];
```

Comment:
I think the second arg here is like a `TempNotifyName` type that matches the second arg for `|tempnotify|` (like how `|uhtml|` works)?

Reply:
Not sure what that last `string` does though. Might be worth it to open some issues in the main PS repo to actually document these.

Reply:
Mia said she'll document these at some point. If we can't figure it out just leave it as `string` and we can tighten up the type later :)